src/development/LocalTest/Services/Authorization/Implementation/ClaimsService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Altinn.Platform.Authorization.Services.Interface;
+using LocalTest.Configuration;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace LocalTest.Services.Authorization.Implementation
+{
+    public class ClaimsService : IClaims
+    {
+        private readonly LocalPlatformSettings _localPlatformSettings;
+
+        public ClaimsService(IOptions<LocalPlatformSettings> localPlatformSettings)
+        {
+            _localPlatformSettings = localPlatformSettings.Value;
+        }
+
+        public Task<List<Claim>> GetCustomClaims(int userId)
+        {
+            var customClaims = new List<Claim>();
+
+            string path = GetCustomClaimsPath(userId);
+
+            if (File.Exists(path))
+            {
+                string content = System.IO.File.ReadAllText(path);
+                Dictionary<string, object> customClaimsJson = (Dictionary<string, object>) JsonConvert.DeserializeObject(content, typeof(Dictionary<string, object>));
+                foreach (var keyValue in customClaimsJson)
+                {
+                    customClaims.Add(new Claim(keyValue.Key, "" + keyValue.Value));
```

Comment:
Right now every custom claim will be added as a string, should this be changed to let the developer specify the claimtype? This would change the json-file from 
```json
{
  "extra:claim": "myclaim",
  "extra:claim2": "myclaim2"
}
```
to something like:
```json
[
  {
    "claimName": "extra:claim",
    "claimValue": 1,
    "claimType": "http://www.w3.org/2001/XMLSchema#integer"
  },
  {
    "claimName": "extra:claim2",
    "claimValue": "2",
    "claimType": "http://www.w3.org/2001/XMLSchema#string"
  }
]
```